3.evaluate_model/README.md

```diff
@@ -2,19 +2,19 @@
 
 In this module, we evaluate the final and shuffled baseline ML models.
 
-After training the final and baseline models in [2.train_model](../2.train_model/), we use these models to predict the labels of the training and testing datasets and evaluate their predictive performance.
+After training the models in [2.train_model](../2.train_model/), we use these models to predict the labels of the training and testing datasets and evaluate their predictive performance.
 
-In [get_model_predictions.ipynb](get_model_predictions.ipynb), we derive the predicted and true phenotypic class for each model and dataset combination.
+In [get_model_predictions.ipynb](get_model_predictions.ipynb), we derive the predicted and true phenotypic class for each model, feature type, and dataset combination.
 These predictions are saved in [compiled_predictions.tsv](predictions/compiled_predictions.tsv).
 
-In [confusion_matrices.ipynb](confusion_matrices.ipynb), we evaluate these 4 sets of predictions with a confusion matrix to see the true/false positives and negatives (see [sklearn.metrics.confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html) for more details).
+In [confusion_matrices.ipynb](confusion_matrices.ipynb), we evaluate these sets of predictions with a confusion matrix to see the true/false positives and negatives (see [sklearn.metrics.confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html) for more details).
 The confusion matrix data are saved to [confusion_matrices](evaluations/confusion_matrices).
 
-In [F1_scores.ipynb](F1_scores.ipynb), we evaluate each model (final, shuffled baseline) on each dataset (train, test, etc) to determine phenotypic and weighted f1 scores.
+In [F1_scores.ipynb](F1_scores.ipynb), we evaluate each model (final, shuffled baseline) trained with each feature type (CP, DP, CP_and_DP) on each dataset (train, test, etc) to determine phenotypic and weighted f1 scores.
```

Comment:
Are these feature types defined anywhere in the documentation? If not, consider providing their definition and/or linking to it.

Reply:
Added to documentation in https://github.com/WayScience/phenotypic_profiling_model/pull/21/commits/7a024fd3f9feb6e0e3a69296cea221d48c5e52c0